### PR TITLE
fix HandleErrors function to not write headers so gzip wrapper can add encoding header

### PR DESCRIPTION
**Pull request by DylanVolz**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/858/
Test PASSed.

File: lib/go-tc/alerts.go [hunk 1]
```diff
@@ -64,8 +65,9 @@ func GetHandleErrorsFunc(w http.ResponseWriter, r *http.Request) func(status int
 			fmt.Fprintf(w, http.StatusText(http.StatusInternalServerError))
 			return
 		}
-		w.Header().Set("Content-Type", "application/json")
-		w.WriteHeader(status)
+
+		w.Header().Set(ContentType, ApplicationJson)
+		w.Header().Set("status", strconv.Itoa(status))
```
Comment (rob05c) [inline lib/go-tc/alerts.go]: Is this going to make errors be returned as `200` by the Wrapper? I think we need to pass the code up (maybe in the Context?), to be finally set by the Wrapper, after gzipping. Unless there's a way to set the HTTP Response Code before writing all the headers

Comment (DylanVolz) [reply lib/go-tc/alerts.go]: you are not mistaken, I looked at the wrong call in my developer tools when I checked the status code.

**Comment by DylanVolz** [comment]
addressing this in a new PR.

**Pull request closed without merging**